app/controllers/users_controller.rb

```diff
@@ -15,16 +17,56 @@ def create
     @user = User.new(user_params)
     if @user.save
       log_in @user
-      flash[:success] = t "user.create.success"
+      flash[:success] = t "users.create.success"
       redirect_to @user
     else
       render :new
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update(user_params)
+      flash[:success] = t("users.edit.success")
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if user&.destroy
```

Comment:
code này có đúng k, tui thu tiền giờ, fix mà ko test
```suggestion
    if @user.destroy
```

mà chỗ này ko check `@user` có phải admin? ko ah